pytorch_lightning/__init__.py

```diff
@@ -20,13 +20,10 @@
 
 if __LIGHTNING_SETUP__:
     import sys
-    sys.stderr.write('Partial import of `torchlightning` during the build process.\n')
+    sys.stdout.write('Partial import of `torchlightning` during the build process.\n')
```

Comment:
is really this needed, just a question...

Reply:
I just checked. The only place where `__LIGHTNING_SETUP__` is set to True is in the docs conf.py. If I remove the if statements in `__init__.py` it does not make a difference in build and tests.

Reply:
It seems it is needed in setup.py for installing the package

Reply:
yes, I mean change from `sys.stderr` to `sys.stdout`

Reply:
aha. no idea xD